src/components/jellyfinActions.ts

```diff
@@ -239,59 +237,45 @@ export function play(state: PlaybackState): void {
  * get PlaybackInfo
  * @param item - item
  * @param maxBitrate - maxBitrate
- * @param deviceProfile - deviceProfile
  * @param startPosition - startPosition
  * @param mediaSourceId - mediaSourceId
  * @param audioStreamIndex - audioStreamIndex
  * @param subtitleStreamIndex - subtitleStreamIndex
  * @param liveStreamId - liveStreamId
  * @returns promise
  */
-export function getPlaybackInfo(
+export async function getPlaybackInfo(
     item: BaseItemDto,
     maxBitrate: number,
-    deviceProfile: DeviceProfile,
```

Comment:
I just realised removing `deviceProfile` is incorrect. I was confused because I couldn't see a parameter with this type on `MediaInfoApiGetPostedPlaybackInfoRequest`.

It's there on the API though and I see it should actually be passed through with `playbackInfoDto`. And using `PlaybackInfoDto` instead of `MediaInfoApiGetPostedPlaybackInfoRequest` here will make it easier to use the generated API.

I'm assuming it would be best to use the types from the SDK models in client code instead of using the types defined for the API itself.